lib/x11utils.pm

```diff
@@ -118,6 +118,7 @@ sub ensure_unlocked_desktop {
         if (match_has_tag('authentication-required-user-settings') || match_has_tag('authentication-required-modify-system')) {
             wait_still_screen;    # Check again as the pop-up may be just a glitch, see bsc#1168979
             if (check_screen([qw(authentication-required-user-settings authentication-required-modify-system)])) {
+                assert_and_click('auth-window-password-prompt');
```

Comment:
This should fail not finding needle in product not getting new desktop? I guess you would need some condition here. Please verify for other products.

Reply:
This `authentication-required-user-settings` or `authentication-required-modify-system` happens very rarely, mainly on S390x which uses xvnc, but recently another bug blocks S390x from proper finish tests.

But yes, I should add system version condition here.